docset/windows/netnat/new-netnat.md

```diff
@@ -56,6 +56,12 @@ PS C:\> New-NetNat -Name "AllTenants" -ExternalIPInterfaceAddressPrefix "a.b.c.0
 This command creates a NAT object named AllTenants for all the computers in the specified subnet.
 This example uses the placeholder a.b.c.0/24 to represent a public Internet address prefix.
 
+From the above, for this example, we'll use 10.0.0.0/24.
+For our example, run the following to setup the NAT network:
```

Comment:
For **this** example, run the following to **set up** the NAT network:

Please do changes as indicated in bold.

Reply:
@johanfreelance9 - commit changes as advised